Sniffs/UpdateURISniff.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace ET\ElegantThemes\Sniffs;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+class UpdateURISniff implements Sniff {
+
+	/**
+	 * Returns the token types that this sniff is interested in.
+	 * @return array(int)
+	 */
+	public function register() {
+		return [ T_COMMENT, T_DOC_COMMENT_STRING ];
+	}
+
+	/**
+	 * Processes the tokens that this sniff is interested in.
+	 *
+	 * @param File $phpcsFile                 The file where the token was found.
+	 * @param int  $stackPtr                  The position in the stack where
+	 *                                        the token was found.
```

Comment:
@lanresmith There are too much space between variable params and their description.

Reply:
Fixed.